src/main/java/no/sysco/middleware/prometheus/kafka/example/Consumer.java

```diff
@@ -0,0 +1,57 @@
+package no.sysco.middleware.prometheus.kafka.example;
+
+import io.prometheus.client.exporter.HTTPServer;
+import io.prometheus.client.hotspot.DefaultExports;
+import no.sysco.middleware.prometheus.kafka.KafkaClientsJmxExports;
+import org.apache.kafka.clients.consumer.ConsumerConfig;
+import org.apache.kafka.clients.consumer.ConsumerRecord;
+import org.apache.kafka.clients.consumer.ConsumerRecords;
+import org.apache.kafka.clients.consumer.KafkaConsumer;
+import org.apache.kafka.clients.producer.ProducerConfig;
+import org.apache.kafka.common.Metric;
+import org.apache.kafka.common.MetricName;
+import org.apache.kafka.common.metrics.KafkaMetric;
+import org.apache.kafka.common.serialization.StringDeserializer;
+
+import java.io.IOException;
+import java.time.Duration;
+import java.util.Collections;
+import java.util.Map;
+import java.util.Properties;
+import java.util.UUID;
+
+public class Consumer {
+
+    public static void main(String[] args) throws IOException {
+        String id1 = UUID.randomUUID().toString();
+        String group1 = UUID.randomUUID().toString();
+
+        final String topic = "topic-in";
+
+        final KafkaConsumer<String, String> kafkaConsumer = new KafkaConsumer<String, String>(getConsumerProps(id1, group1));
```

Comment:
could be enough with `new KafkaConsumer<>...`

Reply:
Later on I will extract all examples in other repository.